packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -382,7 +473,7 @@ $mdc-textfield-disabled-border-on-dark: rgba(white, .3);
 
   .mdc-textfield__label--float-above {
     // NOTE: This is an eyeball'd approximation of what's in the mocks.
-    transform: translateY(calc(-100% - 2px)) scale(.923, .923);
+    transform: translateY(calc(-100% - 2px));
```

Comment:
This change is only affecting the non-box dense textfield. Is that really what's intended, given this PR is about textfield boxes?

Reply:
This is effecting any `dense` textfield whose label has the `mdc-textfield__label--float-above` modifier class. Am I missing something?

Reply:
Also after this monster merge has settled, there will be no `--box` modifier